### PR TITLE
Add Everedit

### PR DESCRIPTION
**Pull request by chirsz-ever**

### CODE REVIEW

**Review by HBelusca — CHANGES_REQUESTED** [review]
Can you show us an example that proves that this application works fine in ReactOS?

**Comment by chirsz-ever** [comment]
VirtualBox_ReactOS-0 4-2010030002

**Review by HBelusca — APPROVED** [review]

File: everedit.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Size = 4.2 MiB
+Category = 7
+URLSite = http://www.everedit.net/
+URLDownload = http://download.everedit.cn/everedit_win32_4480_setup.exe
+SHA1 = 74ef1e92d2337a8beabdd8ac1d387c7674009009
+CDPath = none
+SizeBytes = 4380855
+
+[Section.0804]
+Description = 身躯小巧，性能卓越，自定义功能完善，丰富的主题和脚本，完美的编码、大字符集字符显示，无论您是哪个级别的码农，EverEdit都会给您带来不一样的体验！
```
Comment (learn-more) [inline everedit.txt:15]: This description does not look like the english description at all?

Comment (HBelusca) [reply everedit.txt:15]: :D It looks more "advertisement"-like than the english one :D

**Comment by Zero3K** [comment]
The program is shareware, by the way.

**Comment by chirsz-ever** [comment]
Most files use "Shareware", so I did. But the `ultraiso.txt` uses "Trail version":
https://github.com/reactos/rapps-db/blob/2faffe0b64a44c34309ac0a5f1c1bc5e5290f755/ultraiso.txt#L4

**Pull request merged**